src/app/screens/LNURLPay.tsx

```diff
@@ -130,15 +131,45 @@ function LNURLPay({ details, origin }: Props) {
     }
   }
 
+  function formattedMetadata() {
+    try {
+      const metadata = JSON.parse(details.metadata);
+      return metadata
+        .map(([type, content]: [string, string]) => {
+          if (type === "text/plain") {
+            return ["Description", content];
+          } else if (type === "text/long-desc") {
+            return ["Full Description", <p>{content}</p>];
+          } else if (["image/png;base64", "image/jpeg;base64"].includes(type)) {
+            return [
+              "lnurl",
+              <img src={`data:${type},${content}`} alt="lnurl" />,
+            ];
+          }
+          return undefined;
+        })
+        .filter(Boolean);
+    } catch (e) {
+      console.log(e.message);
```

Comment:
```suggestion
      console.log(e);
```

people don't see the console anyway and for use the whole error might be more useful to debug.

Reply:
Oke, to be more precise I refactored it to using console.error(e);

Reply:
ah, good point. I must remember that for the other similar cases.